tests/test_power.py

```diff
@@ -139,3 +139,12 @@ def test_fractional_buzz_duration(self):
             'duration': 500,
         })
         self.assertIsInstance(msg['buzz']['duration'], int)
+
+    def test_separate_power(self):
+        self.power_board.clear_queue()
+        self.robot.power_board.set_output(PowerOutput.HIGH_POWER_1, True)
```

Comment:
Any chance of looping through all the enums in this test?

Reply:
Not sure what that would actually usefully achieve?